ocaml/xenopsd/lib/xenops_sandbox.ml

```diff
@@ -173,6 +176,10 @@ module Guard (G : GUARD) : SANDBOX = struct
 
   let create ~domid ~vm_uuid path =
     let chroot = chroot ~domid ~vm_uuid in
+    if Sys.file_exists chroot.root then (
```

Comment:
Should this code be moved into the other `Chroot.create` function that creates the new one? Now we are checking the the presence of the directory in two places and react to it.

Reply:
I'll add a parameter to chroot.create so the caller has to be specific whether the existing chroot will be deleted or not, so it can make a specific decision if need be